opensrp-opd/build.gradle

```diff
@@ -72,18 +76,18 @@ dependencies {
     api 'com.android.support:appcompat-v7:28.0.0'
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
-    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
     implementation 'com.android.support:multidex:1.0.0'
 
-    implementation('org.smartregister:opensrp-client-anc:0.0.5-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-anc:0.0.7-SNAPSHOT@aar') {
         transitive = true
         exclude group: 'org.smartregister', module: 'opensrp-client-core'
         exclude group: 'org.smartregister', module: 'opensrp-client-native-form'
         exclude group: 'org.smartregister', module: 'opensrp-client-configurable-views'
         exclude group: 'com.android.support', module: 'appcompat-v7'
+        exclude group: 'net.zetetic', module: 'android-database-sqlcipher'
     }
 
-    api ('org.smartregister:opensrp-client-core:1.8.1-SNAPSHOT@aar') {
+    api ('org.smartregister:opensrp-client-core:1.8.11-SNAPSHOT@aar') {
```

Comment:
Update to the latest core to get the latest fixes from @githengi

Reply:
yes, the latest core should be used